.gitignore

```diff
@@ -58,4 +58,9 @@ view/pages/assets/js/wab.js
 node_modules
 package-lock.json
 
-export/assets
+export/assets/css
+export/assets/fonts
+export/assets/js/wab.js
+export/assets/js/ui.js
+export/assets/js/transpile.js
+
```

Comment:
why are you **not ignoring the entire `export/assets`** directory?

Reply:
The systems and plugin-bagel are needed for the non-es6 to work. I haven't come up with a good way to gem them installed nicely yet. This gets things working until we can untie them. Do you have an alternative to offer?

Reply:
any additional changes to be made here?

Reply:
No changed needed.